vcd/config.go

```diff
@@ -195,68 +195,68 @@ func (cli *VCDClient) getVdcName(d *schema.ResourceData) string {
 // GetOrgAndVdc finds a pair of org and vdc using the names provided
 // in the args. If the names are empty, it will use the default
 // org and vdc names from the provider.
-func (cli *VCDClient) GetOrgAndVdc(orgName, vdcName string) (org govcd.Org, vdc govcd.Vdc, err error) {
+func (cli *VCDClient) GetOrgAndVdc(orgName, vdcName string) (org *govcd.Org, vdc govcd.Vdc, err error) {
```

Comment:
Why it returns only Org as pointer?

Reply:
As mentioned in the corresponding govcd PR, I am replacing structures with pointers as I implement the Get* methods. This function will be modified again when I implement the VDC `Get*`